### PR TITLE
Add help for OAuth config

### PR DESCRIPTION
**Pull request by liggitt**
@deads2k review

### CODE REVIEW

File: architecture/authentication.adoc [hunk 1]
```diff
@@ -240,90 +246,29 @@ To configure how the OAuth server responds to token requests for a client the us
...
+The OAuth server uses a cookie-based session during login and redirect flows.
+
+Master config:
+----
+oauthConfig:
+  sessionConfig:
+    sessionMaxAgeSeconds: 300 <1>
+    sessionName: ssn <2>
+    sessionSecrets: <3>
+    - authentication: &hellip; <4>
```
Comment (deads2k) [inline architecture/authentication.adoc]: Really hoping that's an entity I just don't know....

**Comment by deads2k** [comment]
The current doc won't work at all after yesterday's merges.  I'd merge this in spite of omissions and open issues to restore the external oauth configuration and to fix the session secret stuff.

File: architecture/authentication.adoc (hunk 1 above)
Comment (liggitt) [inline architecture/authentication.adoc]: http://www.w3.org/TR/html4/sgml/entities.html
<!ENTITY hellip   CDATA "&#8230;" -- horizontal ellipsis = three dot leader, U+2026 ISOpub  -->

**Comment by adellape** [comment]
Merging per previous comment + chat with Jordan.

**Pull request merged**